core/ui/EditorToolbar/preview-type-dropdown.tid

```diff
@@ -1,33 +1,55 @@
-title: $:/core/ui/EditorToolbar/preview-type-dropdown
```

Comment:
Hum not sure what happened, that was actually the file I meant to change, in fact it's the only file I altered in my personal tiddlywiki files is $:/core/ui/EditorToolbar/picture-dropdown

Have in mind that I am no coder at all, just was just something I hacked together in the hopes that someone with actual knowledge could possibly improve to a 'commitable state'.